### PR TITLE
made proper documentation

### LINKED ISSUE #26: Create an inline documentation (similar to JSDoc)

### PR DESCRIPTION
**Pull request by subhangi2731**
### What does this implement/fix? Explain your changes.
I have given the proper documentation in README.md and made proper images and also contributing,md and createpr.md and shortest notes.md with proper flow and explanation with being sharp to the project workflow

### Does this close any currently open issues?
 Address #26 

### Any other comments?
I have made all the relevant enhancements and modifications in documentation possible please check out

### CODE REVIEW

File: README.md
Comment (priyanshuone6) [inline README.md:3]: Could you decrease the width and height size so that it becomes equal to the heading

**Comment by subhangi2731** [comment]
yes @priyanshuone6 I will do it please give me some time now I will rectify this

**Comment by subhangi2731** [comment]
@priyanshuone6 please check I have updated the image size

File: README.md
Comment (priyanshuone6) [inline README.md:3]: Could you add a `&nbsp;` element between the image and the text

**Review by priyanshuone6 — CHANGES_REQUESTED** [review]

**Comment by subhangi2731** [comment]
@priyanshuone6 can u check?

**Comment by priyanshuone6** [comment]
Looks good now, thanks

**Comment by subhangi2731** [comment]
Thankyou to you too @priyanshuone6 for suggesting better changes

**Review by priyanshuone6 — APPROVED** [review]

**Comment by subhangi2731** [comment]
@siv2r please review and merge my pr

File: README.md
Comment (siv2r) [inline README.md:47]: since the file location is changed this link is not working. I think `(docs/CreatePR.md)` will fix it. 
more info here: https://stackoverflow.com/questions/11868512/linking-using-relative-path-on-github-readme-md

File: README.md
Comment (siv2r) [inline README.md:46]: Can you use relative path for this link also

**Review by siv2r — CHANGES_REQUESTED** [review]

**Comment by siv2r** [comment]
@subhangi2731 the pr looks good. I made some suggestions. I will merge it after you make to changes :)

**Comment by subhangi2731** [comment]
@siv2r I think the links will be added and working if you merge it or is it before merging also it can work?

**Comment by siv2r** [comment]
@subhangi2731 I checked the links. It is not working. Also, I mentioned the reason why it won't work.
Don't worry. I will merge your PR. Just edit the links :)

**Comment by subhangi2731** [comment]
okay @siv2r I will change the links

**Comment by subhangi2731** [comment]
@siv2r can u check

**Comment by subhangi2731** [comment]
@siv2r please review

File: README.md
Comment (priyanshuone6) [inline README.md:54]: I think you forgot to change this too :)

Comment (subhangi2731) [reply README.md]: @priyanshuone6 can u check now

**Comment by priyanshuone6** [comment]
I think it's ready to merge now

**Comment by subhangi2731** [comment]
@siv2r please merge my pr

**Comment by siv2r** [comment]
@subhangi2731 Great work :tada: :tada:

**Comment by subhangi2731** [comment]
thank u @siv2r

**Pull request merged**
